build.gradle

```diff
@@ -74,15 +86,6 @@ subprojects {
         experimentalRules = false
         disabledRules = []
     }
-
-    shadowJar {
-        zip64 = true
-        archiveFileName = "${archiveBaseName.get()}-${archiveVersion.get()}.jar"
-    }
-}
-
-task allShadowJars {
```

Comment:
and this

Reply:
no, not really, at least as for my checks of the dependencies

Reply:
It was created for cases when we need to build fat jars to put the libraries directly into the repository with the project like [that](https://github.com/arndey/iroha-java-sample/tree/master/app/libs)

Reply:
okay, I'll return these tasks then